libraries/scripting/dependencies-maven/test/kotlin/script/experimental/test/MavenResolverTest.kt

```diff
@@ -81,6 +81,15 @@ class MavenResolverTest : ResolversTestBase() {
         }
     }
 
+    fun testCustomRepositoryId() {
```

Comment:
@ligee I can ignore this test if you wish, but I don't know a way of testing this except using custom repositories

Reply:
Yes, please ignore it, the test below gives a good explanation comment, you can simply copy it. 
I also think that it is a half-good test anyway because for a good one you'd need to test all OS-es separately.
But it is good to have it here anyway, so at least we can test it manually.